.github/workflows/tests.sh

```diff
@@ -8,6 +8,14 @@ verdi daemon start 4
 verdi -p test_${AIIDA_TEST_BACKEND} run .ci/test_daemon.py
 verdi daemon stop
 
+# Make sure
+# - omissions are taken from the top-level .coveragerc
+# - coverage is reported as XML and in terminal including
+#   the numbers/ranges of lines which are not covered,
+# - coverage results of multiple tests are collected,
+# - the base directory for files to consider is aiida/
+export PYTEST_ADDOPTS="--cov-config=.coveragerc --cov-report xml --cov-report term-missing --cov-append --cov=aiida"
+
 verdi -p test_$AIIDA_TEST_BACKEND devel tests
```

Comment:
is this at all included in the coverage report?
This is not using pytest.
Please check how this was run before (probably using `coverage run`)

Reply:
These settings are specific for the `pytest-cov` extension for `pytest` as far as I know?